scripts/tasks/src/ts.ts

```diff
@@ -24,24 +24,26 @@ function prepareTsTaskConfig(options: TscTaskOptions) {
     options.sourceMap = true;
   }
 
-  const { isUsingV8pathAliases, tsConfigFileV8 } = getTsPathAliasesConfigV8();
+  const { isUsingPathAliasesForDx, tsConfigFileForCompilation } = getTsPathAliasesConfigUsedOnlyForDx();
 
-  if (isUsingV8pathAliases) {
-    logger.info(`📣 TSC: V8 package is using TS path aliases. Overriding tsconfig settings.`);
+  if (isUsingPathAliasesForDx) {
+    logger.info(`📣 TSC: Project is using TS path aliases for DX. Disabling aliases for build.`);
     options.baseUrl = '.';
     options.rootDir = './src';
-    options.project = tsConfigFileV8;
+    options.project = tsConfigFileForCompilation;
+
+    return options;
```

Comment:
perf improvement: no need for additional processing in this scenario. exit early